contributors/devel/generating-clientset.md

```diff
@@ -6,7 +6,19 @@ Client-gen is an automatic tool that generates [clientset](../design-proposals/c
 
 The workflow includes three steps:
 
-**1.** Marking API types with tags: in `pkg/apis/${GROUP}/${VERSION}/types.go`, mark the types (e.g., Pods) that you want to generate clients for with the `// +genclient=true` tag. If the resource associated with the type is not namespace scoped (e.g., PersistentVolume), you need to append the `nonNamespaced=true` tag as well.
+**1.** Marking API types with tags: in `pkg/apis/${GROUP}/${VERSION}/types.go`, mark the types (e.g., Pods) that you want to generate clients for with the `// +genclient` tag. If the resource associated with the type is not namespace scoped (e.g., PersistentVolume), you need to append the `// +genclient:nonNamespaced` tag as well.
```

Comment:
so both tags are necessary? The later does not imply the former?

Reply:
@sttts yes, both tags are necessary AFAIK